rocketpy/motors/LiquidMotor.py

```diff
@@ -0,0 +1,390 @@
+            tank = positioned_tank.get("tank")
+            tankPosition = positioned_tank.get("position")
+            totalMass += tank.mass
+            massBalance += tank.mass * (tankPosition + tank.centerOfMass)
+
+        return massBalance / totalMass
+
+    @funcify_method("Time (s)", "Inertia I_11 (kg m²)")
+    def propellant_I_11(self):
+        """Inertia tensor 11 component of the propellant, the inertia is
+        relative to the e_1 axis, centered at the instantaneous propellant
+        center of mass.
+
+        Parameters
+        ----------
+        t : float
+            Time in seconds.
+
+        Returns
+        -------
+        float
+            Propellant inertia tensor 11 component at time t.
+
+        Notes
+        -----
+        The e_1 direction is assumed to be the direction perpendicular to the
+        motor body axis.
+
+        References
+        ----------
+        .. [1] https://en.wikipedia.org/wiki/Moment_of_inertia#Inertia_tensor
+        """
+        I_11 = Function(0)
+        centerOfMass = self.centerOfPropellantMass
+
+        for positioned_tank in self.positioned_tanks:
+            tank = positioned_tank.get("tank")
+            tankPosition = positioned_tank.get("position")
+            I_11 += (
+                tank.inertiaTensor
+                + tank.mass * (tankPosition + tank.centerOfMass - centerOfMass) ** 2
+            )
+
+        return I_11
+
+    @funcify_method("Time (s)", "Inertia I_22 (kg m²)")
+    def propellant_I_22(self):
+        """Inertia tensor 22 component of the propellant, the inertia is
+        relative to the e_2 axis, centered at the instantaneous propellant
+        center of mass.
+
+        Parameters
+        ----------
+        t : float
+            Time in seconds.
+
+        Returns
+        -------
+        float
+            Propellant inertia tensor 22 component at time t.
+
+        Notes
+        -----
+        The e_2 direction is assumed to be the direction perpendicular to the
+        motor body axis, and perpendicular to e_1.
+
+        References
+        ----------
+        .. [1] https://en.wikipedia.org/wiki/Moment_of_inertia#Inertia_tensor
+        """
+        return self.I_11
+
+    @funcify_method("Time (s)", "Inertia I_33 (kg m²)")
+    def propellant_I_33(self):
+        """Inertia tensor 33 component of the propellant, the inertia is
+        relative to the e_3 axis, centered at the instantaneous propellant
+        center of mass.
+
+        Parameters
+        ----------
+        t : float
+            Time in seconds.
+
+        Returns
+        -------
+        float
+            Propellant inertia tensor 33 component at time t.
+
+        Notes
+        -----
+        The e_3 direction is assumed to be the axial direction of the rocket
+        motor.
+
+        References
+        ----------
+        .. [1] https://en.wikipedia.org/wiki/Moment_of_inertia#Inertia_tensor
+        """
+        return 0
```

Comment:
Personally, I believe the main reasoning behind the `funcify` was to avoid making this explicit `Function` "cast". I prefer not returning a `Function` only to apply `funcify` immediately after. Seems a bit redundant.

Reply:
Ok. 
Since you guarantee no problem will be generated by doing this, let's go.